lib/tasks/madeline.rake

```diff
@@ -0,0 +1,11 @@
+namespace :madeline do
+  desc "Enqueue the interest accrual Task. The whenever gem runs this rake task monthly."
```

Comment:
What makes this 'monthly' is the `whenever` command that runs it monthly. Also, what the job actually does is is update all the loans that Madeline currently can access. Should we keep current language describing what it's _for_ or update to describe what it actually does?

Reply:
I don't understand the question, sorry. Can you rephrase/recontextualize?

Reply:
resolved: it is in indeed monthly; no reason to change